pyproject.toml

```diff
@@ -11,9 +11,9 @@ classifiers = ["License :: OSI Approved :: GNU General Public License v3 or late
 dynamic = ["version", "description"]
 dependencies = [
     "numpy~=1.21",
-    "bsb-core~=4.0",
-    "arbor~=0.9",
-    "arborize[arbor]~=4.0"
+    "bsb-core~=4.4",
```

Comment:
Is this new version incompatible with all versions before `4.4`? What new feature or bugfix of 4.4 does  this rely on?